eips_code/eip3670.py

```diff
@@ -18,43 +18,33 @@
     0xf0, 0xf1, 0xf3, 0xf4, 0xf5, 0xfa, 0xfd, 0xfe
 ]
 
-# STOP, RETURN, REVERT, INVALID
-terminating_opcodes = [0x00, 0xf3, 0xfd, 0xfe]
-
-# Only for PUSH1..PUSH32
 immediate_sizes = 256 * [0]
-for opcode in range(0x60, 0x7f + 1):  # PUSH1..PUSH32
-    immediate_sizes[opcode] = opcode - 0x60 + 1
+immediate_sizes[0x60:0x7f + 1] = range(1, 32 + 1)  # PUSH1..PUSH32
```

Comment:
loop was more readable imho

Reply:
It polluted the global scope with `opcode` variable.